### PR TITLE
ItineraryItemSchema_AH_111824

### LINKED ISSUE #1: ItineraryItem Schema

### PR DESCRIPTION
**Pull request by AmandaHockmuth**
Created model folder and added Itinerary Item Schema, installed joi to handle "required" and "date."

### CODE REVIEW

**Review by jgoldenberg29 — CHANGES_REQUESTED** [review]
This schema is looking great! Love the error handling early on. Let's make the change to remove status. Other than that, it looks good to me.

File: models/ItineraryItem.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    location: {
+      address: {
+        type: String,
+        required: [true, "Error: no address provided for itinerary item."],
+      },
+      city: {
+        type: String,
+        required: [true, "Error: no city provided for itinerary item."],
+      },
+      state: {
```
Comment (umavenki) [inline models/ItineraryItem.js:23]: Should we add zip code?

Comment (AmandaHockmuth) [reply models/ItineraryItem.js:23]: Absolutely! Adding

**Review by jgoldenberg29 — APPROVED** [review]
Nicely done. Looks great!

**Review by katsiarynalashcheuskaya — APPROVED** [review]
Great work, Amanda!

**Review by StuvaScript — APPROVED** [review]

**Pull request merged**